packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -213,6 +217,22 @@ export const ChannelView: React.FC = () => {
     }
   }, [currentTabName])
 
+  const collectors: Collector[] = [
+    { assetUrl: 'https://thispersondoesnotexist.com/image', tooltipText: 'Jane', nftsAmount: 4 },
+    {
+      assetUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/channel-posters/2.jpg',
+      tooltipText: 'John',
+      nftsAmount: 2,
+    },
+    { assetUrl: 'https://thispersondoesnotexist.com/image', tooltipText: 'William', nftsAmount: 6 },
+    {
+      assetUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/channel-posters/2.jpg',
+      tooltipText: 'Someone',
+      nftsAmount: 1,
+    },
+    { assetUrl: 'https://thispersondoesnotexist.com/image', tooltipText: 'Someone else', nftsAmount: 7 },
+  ]
+
```

Comment:
Let's maybe set this as an empty array for now and add the `todo` comment here? 
I'm a little worried that we deploy this by accident.

Reply:
Certainly! @eldiegod for now please let's hide this section and also the NFTs tab 🙏

Reply:
its done @kdembler @drillprop